README.md

```diff
@@ -7,26 +7,49 @@ It uses Substrate's smart contract module ‒ the
 [`contracts`](https://github.com/paritytech/substrate/tree/master/frame/contracts)
 pallet.
 
-## Note
-
-This used to be a standalone smart contract node used for the ink! workshop. We
+**Note:** This used to be a standalone smart contract node used for the ink! workshop. We
 have moved the standalone node to [substrate-contracts-node](https://github.com/paritytech/substrate-contracts-node/).
 
-## Installation
+## Quickstart
+
+### Installation 
+
+You need to have executables of both Polkadot, as well as `canvas-node`.
+
+Binary releases can be found here:
+[Polkadot releases](https://github.com/paritytech/polkadot/releases),
+[Canvas releases](https://github.com/paritytech/canvas-node/releases).
+
+Alternatively you can install (or build) the projects yourself:
+
+* `cargo install polkadot-node --git https://github.com/paritytech/polkadot.git --force --locked`
+* `cargo install canvas-node --git https://github.com/paritytech/canvas-node.git --force --locked`
```

Comment:
We need to consider some versioning. Many times the Polkadot repo has new features that might not be compatble with the canvas parachain. E.g. Polkadot v0.9.9 doesn't includes the latest master changes which already integrate a new version of XCM.

Reply:
That's a good point. In the past we always used git tags for this purpose and then added e.g. a `--tag v0.1.0` to those instructions.

A thing that we could think about is to use the same versioning as Polkadot to keep Canvas in lockstep and make it clearer what the compatible version is. So instead of Canvas being on something like v0.1.0, we would make the Canvas version v0.9.9 now as well. 

Wdyt?

Reply:
Yes, that makes sense. I think that is the best way for now.